content/blog/2020-08-05-cml-self-hosted-runners-on-demand-with-gpus.md

```diff
@@ -0,0 +1,183 @@
+---
+title: CML self-hosted runners on demand with GPUs
+date: 2020-08-05
+description: |
+  Use your own GPUs with GitHub Actions & GitLab CI for automated machine learning.
+descriptionLong: |
+  Training models often requires special hardware, like extra memory or GPUs. How can we provide a CI/CD pipeline with this hardware? Find out how to setup your own self-hosted runners on-demand with GPUs for fast training.
```

Comment:
💅 under <80 please

Reply:
still the same problem?

Reply:
@DavidGOrtega PTAL - a minor one

Reply:
@shcheklein 
```
Use your own GPUs with GitHub Actions & GitLab for continuous machine learning.
```

Its 79!

Reply:
https://github.com/iterative/dvc.org/pull/1656/files#diff-348c0fb37e16bfa16cd2c3f0bee054ebR7 - It's def > 80 for me. Am I missing something?

Reply:
Are you speaking about descriptionLong? Sorry I though it was description...

Reply:
so why use the description vs descriptionLong? How long should be description?

Reply:
yes, sorry. it's all about formatting - not the text length per se. Unfortunately prettier does not handle frontmatter automatically.

Reply:
So should I repeat the same text in the description but in the descriptionLong?
Should we add a ticket to include that i docs?

Reply:
It's about formatting - just wrap the text properly with new lines. Does not matter how actually text long is - for both description and descriptionLong - they can be arbitrary long. (obviously, usually, descriptionLong is longer :) ).